### DISCUSSION TITLE
Open .NET would like to join the .NET Foundation

### DISCUSSION THREAD

**Discussion by ghuntley** [dotnet-foundation/Home#74] [Board of Directors]
# overview

The .NET Foundation needs to [resolve problems](https://github.com/dotnet-foundation/Home/discussions/63) related to the articles of incorporation that give Microsoft veto over all decisions before Open .NET joining the foundation would be appropriate, but the desire is to join the .NET Foundation and for the foundation to be open + independent + hold IP of Open .NET once the foundational issues are satisfactorily resolved.

# related issues

- https://github.com/open-dotnet/.github/issues/7
- https://github.com/open-dotnet/.github/issues/44

**Comment by tonyqus** [comment]
I understand that you may have strong concerns on Microsoft hot-reload removal behavior.  That's why Open .NET is created. But this is not Java community. OpenJDK is maintained by a a lot of companies. 

I don't object you creating this project and it's your freedom. But after the creation, please reconsider the workload and the capacity of your team. Are you sure your team can maintain .NET sdk and runtime? It's a very boring job I can imagine. Microsoft hires a few FTE teams to maintain this. At least they gets paid very well. How will you benefits your team? Or finally you are creating geeks for .NET teams and they gets hired by Microsoft. I'm not sure.
